client/src/components/Header.css

```diff
@@ -0,0 +1,34 @@
+.header { 
+    z-index: 3;
+    height: 70px;
+    width: 100%;
+    margin: 0px auto;
+    position: fixed;
+    background: rgba(0, 0, 0, 0.53);
+    transition: all 0.2s ease-in-out;
+}
+
+#text-header{
+    padding-top: 20px;
+    color:white;
+    font-size: 20px;
+    text-decoration: none;
+}
+
+
+.header:hover {
+    background: rgb(0, 0, 0);
+    -webkit-transition: all 0.2s ease-in-out;
+    -moz-transition: all 0.2s ease-in-out;
+    -o-transition: all 0.2s ease-in-out;
+    transition: all 0.2s ease-in-out;
+}
+
+.brand-logo {
+    margin-left: 420px;
+}
```

Comment:
Don't use absolute margins. If you re-scale the page, it'll always be 420 px from the left. Try %